### ISSUE TITLE
creep ability images

### ISSUE THREAD

**Issue by howardchung** [odota/ui#345]
Continuation of https://github.com/odota/core/issues/115

dark_troll_warlord_ensnare
ogre_magi_frost_armor
polar_furbolg_ursa_warrior_thunder_clap
centaur_khan_war_stomp
dark_troll_warlord_raise_dead
enraged_wildkin_tornado
satyr_trickster_purge
satyr_soulstealer_mana_burn 
satyr_hellcaller_shockwave
harpy_storm_chain_lightning
keeper_of_the_light_spirit_form_illuminate_end

@dylanking42 found some of the images, but they need to be pushed to github.  It's kind of messy because we typically source the images from valve CDN (proxied through API_HOST for https)

http://imgur.com/a/vgGAQ

**Comment by vamo89** [comment]
Hey @howardchung, I've started doing that.

For keeper_of_the_light_spirit_form_illuminate_end, I thought about just adding one more point at abilities.json, copying everything from keeper_of_the_light_illuminate_end, because they're identical.

I've also got the new image for mud_golem_hurl_boulder. Not sure why it's not on the list.
hill_troll_priest_heal it's not on the list also, it's because it does not inflict damage? I think it still appears on /casts though.

I just have to assembly the info to build the tooltip and I think it's ready to go.
